libs/components/core/testing/src/shared/input-harness.ts

```diff
@@ -0,0 +1,60 @@
+import { ComponentHarness } from '@angular/cdk/testing';
+
+/**
+ * Harness used to interact with native input elements in tests.
+ * @internal
+ */
+export class SkyInputHarness extends ComponentHarness {
```

Comment:
can we add a host selector to make sure this is constrained to just `input`, `textarea` and `select`?

Reply:
Since this is an internal class, we would need to create a public-facing class that extends this harness. In every case, the public-facing harness would determine what is meant by "control". In the autocomplete harness's case, it's the `[skyAutocomplete]` attribute.